lua/vuffers/ui.lua

```diff
@@ -49,7 +52,9 @@ local function _generate_line(buffer)
   local pinned_icon = config.get_view_config().pinned_icon
   local pinned_icon_text = bufs.is_pinned(buffer) and pinned_icon .. " " or ""
   local icon_text = icon ~= "" and icon .. " " or "  "
-  local text = pinned_icon_text .. icon_text .. buffer.name
+  local filename_text = config.get_view_config().show_file_extension and _get_buffer_name_with_extension(buffer)
```

Comment:
I think the display name (`buffer.name`) should include the extension if config is set so. And UI doesn't need to care about config.
It could be done [here](https://github.com/Hajime-Suzuki/vuffers.nvim/blob/515ce9c1b31154acaf9d54c2441865b83f51162c/lua/vuffers/buffers/buffer-utils.lua#L95)